WooCommerce/Classes/ViewRelated/Products/Cells/Product Images/ProductImagesHeaderTableViewCell.xib

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="15505" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES">
-    <device id="retina6_1" orientation="portrait" appearance="light"/>
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="18122" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES">
```

Comment:
If I understand correctly, `ProductImagesHeaderTableViewCell` isn't being modified anymore, so maybe this change can be reverted? The diff looks like Xcode just messing around with XIB files with no meaningful changes

Reply:
Thanks! Yes for all 3 `ProductFormViewController.xib`, `ProductImagesHeaderTableViewCell.xib`, and `TitleAndTextFieldTableViewCell.xib` should return as they were before the initial changes on the top of the images. I wasn't 100% sure how to do it without polluting further the commit history, just reverting the files separately should do, right ?

Reply:
> just reverting the files separately should do, right?

If that's possible then it sounds like a good option. Personally I don't worry too much about commit history these days, as I see the PR as the unit of work. To be honest, I thought our repo was set up to squash and merge by default, but it seems it's not 😬 I would probably squash/merge this PR since the final approach is completely different than the initial one, and it's a pretty self-contained change, but I have no strong preference either way.